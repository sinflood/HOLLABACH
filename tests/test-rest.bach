inst Piano{
[C3/4 R C3/4 R]
}
