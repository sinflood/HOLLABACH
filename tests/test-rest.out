Timing Resolution (pulses per quarter note)
4

Instrument,0,,

Tick,Note(0-127),Velocity(0-127),Length,
0,48,80,8
8,48,80,8
